### DISCUSSION TITLE
I can help

### DISCUSSION THREAD

**Discussion by noraj** [CervantesSec/cervantes#14] [Show and tell]
As you can see I tested Cervantes and opened a bunch of issues accordingly.
I have a long experience with collaborative penetration test, vulnerability database and reporting platform, I tested nearly all of them and indexed them here https://inventory.raw.pm/tools.html#title-tools-collaboration-report.
I also used some on a day to day basis at work, so if you need advices on architecture/design, ideas, feedback etc. just ask me I'll be happy to help.
I will also add Cervantes to the Inventory soon. 😉

**Comment by mesquidar** [comment]
Hi @noraj , thank you appreciate your feedback.

If you have more feedback, ideas or suggestions are welcomed :) 

Have a nice day
Thank you :)
